content/src/roadmaps/license-tasks/elevator-registration.md

```diff
@@ -4,7 +4,7 @@ webflowId: "65d91b5ef6f946f6f6e0fbb2"
 urlSlug: "elevator-registration"
 name: "Register Your Elevator Device"
 webflowName: "Elevator Registration"
-filename: "elevator-registration_1"
```

Comment:
Micah confirmed this _1 was accidental, don't think there's any repercussions to fixing it here

Reply:
Agreed. There shouldn't be.